### DISCUSSION TITLE
Support for overseerr

### DISCUSSION THREAD

**Discussion by thomaslochet** [hrfee/jfa-go#121] [Ideas]
It will be so cool if a support for overseerr is added

Overseerr is an open source alternative for Ombi
https://github.com/sct/overseerr

**Comment by hrfee** [comment]
~~I'll look into this now.~~ It doesn't look like Overseerr works with Jellyfin at all, apart from a fork for a PR which has since been closed. I'll look into this again if it ever supports Jellyfin.

**Reply by caioolivv** [reply]
[Jellyseerr](https://github.com/Fallenbagel/jellyseerr), an Overseerr fork for Jellyfin, has been working really well!

**Reply by binarygeek119** [reply]
Also works with emby with a environmental argument

**Comment by devanteweary** [comment]
Just came here today because I was gonna mention it! But for Jellyseerr as well (what I use).
